pages/partials/question.ejs

```diff
@@ -75,11 +75,24 @@
         <% } else { %>
         <div class="row">
           <div class="col-sm-6">
-            <% if (showGradeButton) { %><button class="btn btn-primary" name="__action" value="grade">Grade</button><% } %>
+              <% if (showGradeButton) { %>
+                  <% if (question_context == 'student_homework') { %>
+                      <button class="btn btn-primary question-grade" name="__action" value="grade">Grade
+                          <% if ((assessment_question.tries_per_variant - variant.num_tries) > 1) { %>
+                              <span class="badge">
+                                  <%= String(assessment_question.tries_per_variant - variant.num_tries) %>
+                                  tries left
+                              </span>
+                          <% } %>
+                      </button>
+                  <% } else { %>
+                      <button class="btn btn-primary question-grade">Grade</button>
```

Comment:
This is the reason for the `unknown __action: undefined` bug. The Grade button needs to look like the above one to work:
```html
<button class="btn btn-primary question-grade" name="__action" value="grade">
```